akamai/src/main/java/org/frankframework/extensions/akamai/NetStorageCmsSigner.java

```diff
@@ -39,14 +39,14 @@ public class NetStorageCmsSigner {
 	protected static final String ACTION_HEADER = "X-Akamai-ACS-Action";
 	protected static final String AUTH_DATA_HEADER = "X-Akamai-ACS-Auth-Data";
 	protected static final String AUTH_SIGN_HEADER = "X-Akamai-ACS-Auth-Sign";
-	private static final Random RANDOM = new Random();
+	private static final SecureRandom RANDOM = new SecureRandom();
```

Comment:
Deze kan je beter een maal (centraal) aanmaken in plaats van op verschillende plaatsen.

Reply:
Ja dat kan natuurlijk, maar voelt een beetje als micro-optimalisatie.. Het gaat hier om een klein object.

Reply:
https://codeql.github.com/codeql-query-help/java/java-random-used-once/ 😄

Reply:
Doe ik.